queries.sql

```diff
@@ -1 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
+
+SELECT * FROM animals WHERE name LIKE '%mon%';
```

Comment:
- [ ] It would be better if we change `%mon%` to `%mon` instead. This will help us make the right queries :100:

Reply:
@addod19 Thanks! I will correct that.

Reply:
> * [ ]  It would be better if we change `%mon%` to `%mon` instead. This will help us make the right queries 💯 It is strange it actually outputs the same result. `%mon%` means starts and ends in `mon` while `%mon` means ends with `mon` but from the screenshot. It outputs the same result, quite strange 🤔

Yes, the same result. Thanks, really! Totally new lesson for me. just awarded now!